tests/core/test_block_manager.py

```diff
@@ -595,3 +595,43 @@ def test_sliding_window_multi_seq():
 
     # assert all blocks are free now
     assert block_manager.get_num_free_gpu_blocks() == num_gpu_blocks
+
+
+def test_mark_blocks_as_computed_with_prefix_cache_and_chunked_prefill():
```

Comment:
do we have corresponding test in v2?

Reply:
We don't need to test v2 because v2 automatically mark touched blocks as computed.